### PR TITLE
refactor: Restructure "js" directory

### PR DESCRIPTION
**Pull request by bennyn**
This PR gets rid off the "js" directory. It also disables hard code coverage thresholds because we cannot combine coverage reports from main & renderer process yet (see https://github.com/istanbuljs/nyc/issues/1011).

### CODE REVIEW

File: electron/src/util/OriginValidator.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as url from 'url';
```
Comment (ffflorian) [inline electron/src/util/OriginValidator.ts]: Missing copyright header

File: electron/src/window/AboutWindow.ts [hunk 2]
```diff
@@ -142,4 +147,4 @@ const showWindow = () => {
   aboutWindow.show();
 };
 
-export {showWindow};
+export default {showWindow};
```
Comment (ffflorian) [inline electron/src/window/AboutWindow.ts]: I thought we agreed to not use `export default` anymore :wink:

File: electron/src/window/AboutWindow.ts [hunk 3]
```diff
@@ -18,13 +18,13 @@
  */
 
 import {BrowserWindow, IpcMessageEvent, app, ipcMain, session, shell} from 'electron';
-import fileUrl = require('file-url');
 import * as path from 'path';
 
-import {i18nLanguageIdentifier} from '../interfaces/';
+import fileUrl = require('file-url');
```
Comment (ffflorian) [inline electron/src/window/AboutWindow.ts]: This should work, too:

```suggestion
import fileUrl from 'file-url';
```

**Comment by bennyn** [comment]
Thanks! Fixed.

**Pull request merged**